Casks/f/feedflow.rb

```diff
@@ -0,0 +1,28 @@
+cask "feedflow" do
+  version "1.0.77"
+  sha256 :no_check
+
+  url "https://github.com/prof18/feed-flow/releases/download/#{version}-desktop/FeedFlow-#{version}.dmg",
+      verified: "github.com/prof18/feed-flow/"
+  name "FeedFlow"
+  desc "Minimalistic RSS Reader"
```

Comment:
```suggestion
  desc "RSS reader"
```

The description should be concise (see [documentation](https://docs.brew.sh/Cask-Cookbook#stanza-desc)).